acq4/modules/MultiPatch/multipatchTemplate.py

```diff
@@ -76,9 +76,6 @@ def setupUi(self, MultiPatch):
         self.slowBtn.setCheckable(True)
```

Comment:
This file is automatically generated from `multipatchTemplate.ui`. You should make your changes there (run `designer` to edit the .ui file), and I'll show you how to regenerate this file correctly.